lib/v2/zhonghuawang/maintainer.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    '/news/military': ['jiaaoMario'],
+    '/news': ['jiaaoMario'],
```

Comment:
This should match with `router.js`
```suggestion
    '/news/:category?': ['jiaaoMario'],
```